app/assets/stylesheets/conversations.scss

```diff
@@ -180,6 +180,11 @@
     a:hover {
       text-decoration: none;
     }
+    .pagination {
```

Comment:
@SansPseudoFix We have no SCSS styleguide yet. @houndci seems to ignore our hound config sometimes. I think that might be after a forced push. You don't have to change this if you don't want to.